torch/csrc/jit/import.cpp

```diff
@@ -191,7 +228,8 @@ std::shared_ptr<script::Module> load(std::istream& in) {
   };
 
   ScriptModuleDeserializer deserializer(&in);
-  deserializer.deserialize(module_lookup);
+  // TODO: add device support in C++ API
```

Comment:
Wouldn't c10::nullopt be a better default?

Reply:
Using 'cpu' to reserve the original behavior of cpp API. (originally, we restore everything to CPU), otherwise some cases in test_api will break. Will change it to https://github.com/pytorch/pytorch/pull/14711